models/contacts.js

```diff
@@ -1,19 +1,66 @@
-// const fs = require('fs/promises')
+const fs = require('fs/promises')
+const path = require('path')
+const Joi = require('joi')
 
-const listContacts = async () => {}
+const contactsPath = path.join(__dirname, '..', 'db', 'contacts.json')
```

Comment:
invalid path to file. You have database in models folder, not db. Why not just 
`const contactsPath = path.join(__dirname, 'contacts.json')` ?